### DISCUSSION TITLE
18BEE6B4-AFE0-5F33-9B86-F7039DC54ADC

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#31] [General]
# Cygnus X-3

Cygnus X-3 is a high-mass X-ray binary (HMXB) located about 24,124±3,586 light-years away in the constellation Cygnus. It is often considered to be a microquasar, and it is believed to be a compact object in a binary system which is pulling in a stream of gas from an ordinary star companion.

(NOTE: Black hole addons tend to be performance heavy and might not work in all devices)

https://celestia.mobi/addon?item=18BEE6B4-AFE0-5F33-9B86-F7039DC54ADC&lang=en

**Comment by Dirgham2015** [comment]
:O

**Comment by ancientrocket508** [comment]
DIS COOL

**Comment by i22a** [comment]
Magic

**Comment by curnyloveautentik-maker** [comment]
This can't be like a problem. Blue, like a black hole.

**Comment by curnyloveautentik-maker** [comment]
Note: But we have it as it was without black holes, so why, more importantly, as it was, the stars, no part of the universe
